src/state/session/index.tsx

```diff
@@ -658,11 +563,14 @@ export function Provider({children}: React.PropsWithChildren<{}>) {
       resumeSession,
       removeAccount,
       selectAccount,
-      updateCurrentAccount,
+      refreshSession,
       clearCurrentAccount,
     ],
   )
 
+  // as we migrate, continue to keep this updated
+  __globalAgent = agent
+
```

Comment:
@gaearon special note of this

Reply:
Should we do a pass to use a hook first? And then replace the implementation.